src/css.js

```diff
@@ -113,6 +113,13 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableDimensionsInFullscreenMode && window.top !== window.self &&
+		document.msFullscreenElement ) {
+		val = Math.round( elem.getBoundingClientRect()[ name ] * 100 );
```

Comment:
Hmm, now that I think about it... Since we currently use `elem.offsetWidth` & `elem.offsetHeigh` that are non-fractional in all browsers, we should use the same methods here. IOW, you can just assign:
````js
val *= 100;
```
as you already have the value, it's just 100 times too small. I'll most likely change it to `getBoundingClientRect` later to get fractional values but I'll do it for all browsers at once, it should consistently be integers before that happens.